tests/common.go

```diff
@@ -3629,7 +3631,7 @@ func DeleteClusterWithUID(name string, uid string, orgID string, ctx context1.Co
 	if err != nil {
 		return err
 	}
-	err = backupDriver.WaitForClusterDeletionWithUID(ctx, name, uid, orgID, clusterDeleteTimeout, clusterCreationRetryTime)
+	err = backupDriver.WaitForClusterDeletionWithUID(ctx, name, uid, orgID, backupDeleteTimeout, backupDeleteRetryTime)
```

Comment:
Why did we change this?

Reply:
Since we are setting delete backups to true during cluster deletion it can take more time for deletion.
As discussed changed the name convention to clusterDeleteTimeout, clusterDeleteRetryTime.